livingatlas/scripts/la-pipelines

```diff
@@ -26,7 +26,12 @@ set -e # Stop on any failure
 if [[ $SCRIPTPATH == "/usr/bin" ]] ; then PROD=true ; else PROD=false ; fi
 
 if [[ $PROD = false ]]; then
-    VER=$(grep -oPm1 "(?<=<version>)[^<]+" "../pom.xml")
+  # grep with -P option doesnt work on Mac OSX - perl regex arent supported,
+  if [[ "$OSTYPE" == "darwin"*  ]]; then
+    VER=$(grep version ../pom.xml | grep -v '<?xml'| head -1 | sed 's/[[:space:]]//g' | sed -E 's/<[/]{0,1}version>//g')
```

Comment:
The current version of `la-pipelines` fails for me in ubuntu because of this (`VER` is empty). This works for me:
```
diff --git a/livingatlas/scripts/la-pipelines b/livingatlas/scripts/la-pipelines
index 9146b3e9..f5c78fa3 100755
--- a/livingatlas/scripts/la-pipelines
+++ b/livingatlas/scripts/la-pipelines
@@ -26,12 +26,7 @@ set -e # Stop on any failure
 if [[ $SCRIPTPATH == "/usr/bin" ]] ; then PROD=true ; else PROD=false ; fi
 
 if [[ $PROD = false ]]; then
-  # grep with -P option doesnt work on Mac OSX - perl regex arent supported,
-  if [[ "$OSTYPE" == "darwin"*  ]]; then
     VER=$(grep version ../pom.xml | grep -v '<?xml'| head -1 | sed 's/[[:space:]]//g' | sed -E 's/<[/]{0,1}version>//g')
-  else
-    VER=$(grep -om1 "(?<=<version>)[^<]+" "../pom.xml")
-  fi
 else
     VER=$(dpkg-query --show -f '${Version}' la-pipelines)
 fi
```

but if for some reason you want to maintain the `if darwin` , just add the `-P` again:
```
diff --git a/livingatlas/scripts/la-pipelines b/livingatlas/scripts/la-pipelines
index 9146b3e9..2412f5b9 100755
--- a/livingatlas/scripts/la-pipelines
+++ b/livingatlas/scripts/la-pipelines
@@ -30,7 +30,7 @@ if [[ $PROD = false ]]; then
   if [[ "$OSTYPE" == "darwin"*  ]]; then
     VER=$(grep version ../pom.xml | grep -v '<?xml'| head -1 | sed 's/[[:space:]]//g' | sed -E 's/<[/]{0,1}version>//g')
   else
-    VER=$(grep -om1 "(?<=<version>)[^<]+" "../pom.xml")
+    VER=$(grep -oPm1 "(?<=<version>)[^<]+" "../pom.xml")
   fi
 else
     VER=$(dpkg-query --show -f '${Version}' la-pipelines)
```

Reply:
thanks @vjrj  - i removed the check. agree, much nicer without it !